js/editor-ta.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * @constructor
+ * @param {DOM} elementId
+ * @param {Settings} settings
+ */
+function EditorTextArea(editorElement, settings) {
+  this.element_ = editorElement;
+  this.settings_ = settings;
+  this.numLines_ = 1;
+
+  this.setTheme();
+
+  const initFontSize = this.settings_.get('fontsize') + 'px';
+
+  this.textarea_ = document.createElement('textarea');
+  this.textarea_.setAttribute('id', 'editor-ta');
+  this.textarea_.style.fontSize = initFontSize;
+  this.textarea_.addEventListener('input', function() {
+    this.onChange();
+  }.bind(this));
+  window.addEventListener('resize', function() {
```

Comment:
When is this removed?

Reply:
never! I'll update the editor API to include a destruction method.